parm/aero/berror/aero_diagb.yaml.j2

```diff
@@ -35,6 +35,14 @@ background error:
   filename_trcr: stddev.fv_tracer.res.nc
   filename_cplr: stddev.coupler.res
 
+Climate background error stddev:
+  filetype: fms restart
+  skip coupler file: true
+  datapath: Data/staticb/
```

Comment:
Make paths consistent

Reply:
> Make paths consistent

I need to think about this a little bit. In the past the staticb directory was defined by the workflow . Should we continue that? Or have it be in ./staticb?  I just don't want it to get mixed up with the hybrid stddev which will be in ./stddev

Reply:
We could call it something like clim_stddev, but regardless it shouldn't be Data/... I don't think